runtime/cudaq/qis/managers/BasicExecutionManager.h

```diff
@@ -110,14 +110,6 @@ class BasicExecutionManager : public cudaq::ExecutionManager {
     if (!executionContext)
       return;
 
-    if (isInTracerMode()) {
```

Comment:
I believe this diff is from PR #1457, so presumably it will go away once that PR is merged?

Reply:
Yea that's right. I just needed the change to keep the tests passing locally.